web/src/client/mixins.js

```diff
@@ -26,14 +26,14 @@ const withDBus = {
    * @param {string} iface - D-Bus iface
    * @return {Object} a cockpit DBusProxy
    */
-  async proxy(iface) {
+  async proxy(iface, path) {
```

Comment:
Please, document the `path` argument. And do not forget to mention that you can leave it undefined.